src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_core.clj

```diff
@@ -24,6 +24,9 @@
 (def ContextHandlerOptions
   (dissoc (merge jetty9-core/ContextHandlerOptions RouteOption) :server-id))
 
+(def WebsocketHandlerOptions
+  (dissoc (merge jetty9-core/CommonOptions RouteOption) :server-id))
```

Comment:
@cprice404 There is in the `jetty9-core` namespace, but the set of common options for the webserver service and the webrouting service are different, since the webrouting service also needs to support the `:route-id` option in addition to the `:redirect-if-no-trailing-slash` option.